server/api/operator.go

```diff
@@ -339,6 +340,32 @@ func (h *operatorHandler) Delete(w http.ResponseWriter, r *http.Request) {
 	h.r.JSON(w, http.StatusOK, "The pending operator is canceled.")
 }
 
+// @Tags operator
+// @Summary lists the finished operators in the given time.
+// @Param from query integer false "From Unix timestamp"
+// @Produce json
+// @Success 200 {object} []operator.OpRecord
+// @Failure 400 {string} string "The request is invalid."
+// @Failure 500 {string} string "PD server failed to proceed the request."
+// @Router /operators/records [get]
+func (h *operatorHandler) Records(w http.ResponseWriter, r *http.Request) {
+	var from time.Time
+	if fromStr := r.URL.Query()["from"]; len(fromStr) > 0 {
+		fromInt, err := strconv.ParseInt(fromStr[0], 10, 64)
+		if err != nil {
+			h.r.JSON(w, http.StatusBadRequest, err.Error())
+			return
+		}
+		from = time.Unix(fromInt, 0)
+	}
+	records, err := h.GetRecords(from)
```

Comment:
The signature of this function argument is `start`, what about keeping them the same?

Reply:
fix, all signature should be `from`.